menas/pom.xml

```diff
@@ -391,6 +386,16 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <version>${maven.rat.plugin.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>ui/3rdParty/cRonstrue/cronstrue.min.js</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
```

Comment:
Why is it that it was not satisfactory to put the _exlude_ path in the list of _exlucde_ paths we have in the root `pom.xml` already?

(I saw a discussion on WA about this, but it was not clear)

Reply:
Because the check is done in the parent and then with each module (with path relative to the module root). And this new file belongs to_Menas_ module. So the configuration had to be done within the affected module.
(I guess the relative patch could have been in the parent `pom.xml` as well. But that would be: first confusing, second potentially exclude unwanted files in other modules.)

Reply:
Well, I am saying that because it seems that some of the exclusions we have in the root config are also sometimes actually subpackage specific, e.g. `**/node_modules/**`, `**/_site/**`, `**/ui/dist/**`. So perhaps we could rethink those, too.

Reply:
Oh, them having wildcards, I didn't realize. Are these all, or should I check more?

Reply:
Moved `**/node_modules/**`, other is not present.

Reply:
The ones I listed were the only sort of non-general cases. So IMHO only those were actually in question.